### PR TITLE
Add Editor icons and adjust default icon size

### PR DESCRIPTION
**Pull request by sheridanvk**
This PR adds the icons we need to use the shared Icon component in the editor, and adjusts the default size of the icon to 1em. The Community site has overridden the size of the icon to match this new default size already, so this will allow us to remove those overrides.

Icons were added to https://shared-components.glitch.me/#StoryIcon

Remix for testing: https://atlantic-spot.glitch.me/#StoryIcon

### CODE REVIEW

File: .glitch-assets [hunk 1]
```diff
@@ -58,7 +58,6 @@
 {"name":"mouse.png","date":"2019-09-10T14:07:50.101Z","url":"https://cdn.glitch.com/3e774f68-7017-4c21-bced-dc0e98e90643%2Fmouse.png","type":"image/png","size":2214,"imageWidth":32,"imageHeight":32,"thumbnail":"https://cdn.glitch.com/3e774f68-7017-4c21-bced-dc0e98e90643%2Fmouse.png","thumbnailWidth":32,"thumbnailHeight":32,"uuid":"n48qBPsfGdKgWFTM"}
 {"name":"party.png","date":"2019-09-18T14:12:56.192Z","url":"https://cdn.glitch.com/967447b4-a24e-4bae-80e4-3aee75414e03%2Fparty.png","type":"image/png","size":840,"imageWidth":32,"imageHeight":32,"thumbnail":"https://cdn.glitch.com/967447b4-a24e-4bae-80e4-3aee75414e03%2Fparty.png","thumbnailWidth":32,"thumbnailHeight":32,"uuid":"TWHdhjQQDEgfyFmG"}
 {"name":"handshake.png","date":"2019-09-18T14:12:56.216Z","url":"https://cdn.glitch.com/967447b4-a24e-4bae-80e4-3aee75414e03%2Fhandshake.png","type":"image/png","size":388,"imageWidth":32,"imageHeight":32,"thumbnail":"https://cdn.glitch.com/967447b4-a24e-4bae-80e4-3aee75414e03%2Fhandshake.png","thumbnailWidth":32,"thumbnailHeight":32,"uuid":"TT0ywpqxKZe7TjvN"}
-<<<<<<< HEAD
```
Comment (sheridanvk) [inline .glitch-assets]: Seems like this came in from an old merge conflict, so I removed it

**Comment by whimsicallyson** [comment]
all makes sense to me! are you good to deploy and update the module, or would you like a buddy?

**Comment by sheridanvk** [comment]
thank you! will give it a go tomorrow and let you know if I have any issues
:)

**Comment by keithk** [comment]
:rocket: PR was released in [`v0.19.0`](https://github.com/glitchdotcom/shared-components/releases/tag/v0.19.0) :rocket:

**Pull request merged**